README.md

```diff
@@ -1 +1,15 @@
-# igniteui-angular-i18n
\ No newline at end of file
+# igniteui-angular-i18n
+
+[![Build Status](https://travis-ci.org/IgniteUI/igniteui-angular-i18n.svg?branch=master)](https://travis-ci.org/IgniteUI/igniteui-angular-i18n)
+
+## Purpose
+
+This respository stores the Japanese and Korean resource strings for the Ignite UI for Angular `i18n`. The English resource strings are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository. It also contains the interfaces for each localizable component's `resourceStrings` property. They can be used as a schema for creating custom resource strings.
```

Comment:
This repo may contain other languages in the future, not only JA and KO.

Reply:
`It also contains the interfaces for each localizable component's resourceStrings property.`  - this is no longer valid. You could put a link to the interfaces location from igniteui-angular repo.